docs/UserGuide.adoc

```diff
@@ -393,6 +393,19 @@ or to clear the cart after completing the purchase.
 |Clears the cart of all items.
 |===
 
+==== Move all ingredients currently in the cart to inventory
+This command allows you to move all the ingredients from the cart into your inventory. This command also
```

Comment:
I think can add a reason why a user may want to move all the ingredients to the inventory